assets/src/components/supported-templates-toggle/index.js

```diff
@@ -9,20 +9,32 @@ import { __ } from '@wordpress/i18n';
  */
 import { Options } from '../options-context-provider';
 import { AMPSettingToggle } from '../amp-setting-toggle';
+import { AMPNotice } from '../amp-notice';
 
 export function SupportedTemplatesToggle() {
 	const { editedOptions, updateOptions } = useContext( Options );
 
-	const { all_templates_supported: allTemplatesSupported, theme_support: themeSupport } = editedOptions;
+	const { all_templates_supported: allTemplatesSupported, reader_theme: readerTheme, theme_support: themeSupport } = editedOptions;
 
-	return [ 'standard', 'transitional' ].includes( themeSupport ) ? (
-		<AMPSettingToggle
-			checked={ true === allTemplatesSupported }
-			text={ __( 'This will allow all of the URLs on your site to be served as AMP by default.', 'amp' ) }
-			title={ __( 'Serve all templates as AMP regardless of what is being queried.', 'amp' ) }
-			onChange={ () => {
-				updateOptions( { all_templates_supported: ! allTemplatesSupported } );
-			} }
-		/>
-	) : null;
+	const isReaderMode = 'reader' === themeSupport;
+	const isLegacy = isReaderMode && 'legacy' === readerTheme;
+
+	return (
+		<>
+			{ isLegacy && (
+				<AMPNotice>
```

Comment:
There seems to be an alignment issue here:

![image](https://user-images.githubusercontent.com/134745/87082139-608de080-c1df-11ea-8316-7ade3d8824e1.png)

Reply:
Fixed in f6c407c4cb7e93a30e816bed1586d010b1b35c28